pkg/filesystem/subvolume.go

```diff
@@ -34,10 +33,17 @@ type fsStruct struct {
 }
 
 type subVolumeInfo struct {
-	svg string
-	fs  string
+	svg   string
+	fs    string
+	state string
 }
 
+const (
+	InUse             = "in-use"
+	Stale             = "stale"
+	StaleWithSnapshot = "stale-with-snapshot"
```

Comment:
no need to be exported variables as they are not used anywhere else outside this file?